.github/workflows/build.yml

```diff
@@ -29,3 +28,28 @@ jobs:
         uses: codecov/codecov-action@v1.0.14
         with:
           file: ./coverage.txt
+      - name: Get image repository
+        run: echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+      - name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
```

Comment:
Same question as above

Reply:
This step is optional for sure you may customize build step as you like with this list of platforms.